test/fixture-download/local-with-exclude-dir/integration-env/gce/module-gce-b/terragrunt.hcl

```diff
@@ -0,0 +1,11 @@
+inputs = {
+  name = "Module GCE B"
+}
+
+terraform {
+  source = "../../../..//hello-world-no-remote"
+}
+
+dependencies {
+  paths = ["../../aws/module-aws-a"]
```

Comment:
Looks like all these `module-gce` configs depend on `aws`. I know this was the same as before, but is that intentional?

Reply:
I think these were written by a contributor. Don't know if intentional or not, but probably not worth tweaking as part of this already massive PR.